base/applications/regedit/lang/sk-SK.rc

```diff
@@ -1,5 +1,4 @@
-/* TRANSLATOR : Mário Kačmár /Mario Kacmar/ aka Kario (kario@szm.sk)
- * DATE OF TR.: 07-07-2008
+/* TRANSLATOR: 2008 Mario Kacmar <kario@szm.sk>
```

Comment:
Is making the name ASCII intentional?

Reply:
yes it was. we did that in some other locations as well for Mario Kacmar. in some cases even just "Kario" was used. I don't strictly insist on that change though, in case somebody really would feel offended by it.

Reply:
The preference nowadays is to use utf-8 encoding, so the accents are allowed.
Cc @Venca24

Reply:
Allowed yes, but why is it mandatory here? if nobody gives a fuck about::
![image](https://github.com/user-attachments/assets/c59241d4-e0f1-4f8c-9712-e6ba1e6ab120)
![image](https://github.com/user-attachments/assets/81662a91-d1b0-48b1-acb5-1506b0f13890)
![image](https://github.com/user-attachments/assets/86ea5d28-dcfb-4ca7-a3ca-2bedab125482)
![image](https://github.com/user-attachments/assets/d21d2f2b-26b8-4423-8297-5e30e50d95ca)
![image](https://github.com/user-attachments/assets/72f19d24-c7e0-422d-b98c-82b20e6cb614)
![image](https://github.com/user-attachments/assets/142112c8-fa3d-40ed-9cd1-5149ee4c4dec)

Reply:
These are being changed to the correct diacritics when @Venca24 is updating these translations.
Here in regedit, they are already present, and you needlessly remove them.

Reply:
I don't want to endlessly discuss this, so here is the Hermes-muter https://github.com/reactos/reactos/pull/7193/commits/ceb8ad59c8e0b3b616d58c18517f72dc970a7c1b
Resolved.